### PR TITLE
Submitting Evaluation for Teammates

### PR DESCRIPTION
**Pull request by pmets**
Here is my evaluation

### CODE REVIEW

File: teammates_evaluation.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+---
+
+## Project Data
+
+1. Project description: <br>
+<!--
+What is the purpose of this project? What does the code do? What type of users
+does it have?
+-->
+	TEAMMATES is an online tool for managing peer evaulations and other feedback for students. It is a cloud based service for teachers and students.
```
Comment (joannakl) [inline teammates_evaluation.md]: this line is formatted as code

File: teammates_evaluation.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+## License
+
+1. What is the project's license? <br>
+<!--
+In most repositories there will be a file named LICENSE or something similar in
+the root level of the repository. This is the one to examine. There may be
+different licenses on specific files, but the project will have a main license.
+-->
+	GNU General Public License
```
Comment (joannakl) [inline teammates_evaluation.md]: formatted as code

**Comment by pmets** [comment]
Resolved the formatting issues!

**Comment by joannakl** [comment]
Thanks for the review.

**Pull request merged**